src/client/components/MessageForm/SendMessageForm.js

```diff
@@ -27,14 +27,19 @@ function SendMessageForm({ channelId, userId }) {
   return (
     <>
       <form onSubmit={handleSubmit} className="send-new-message-form">
-        <img
-          src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
-          alt=""
-        />
-        <input
-          value={input}
-          onChange={(event) => setInput(event.target.value)}
-        />
+        <div className="new-message-form-imag">
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
```

Comment:
Why are you using this image? It looks nothing like the image in the mockup

Reply:
I did not add any image . It was there like that. but i fixed its flexible size to screen size.

Reply:
I agree . It should not be there . instead there should be file upload system and its Icone.

Reply:
![Screenshot 2020-10-06 at 10 00 35](https://user-images.githubusercontent.com/54095122/95176018-90d42380-07bc-11eb-8b2b-1934e121633f.png)